packages/react-scripts/README.md

```diff
@@ -97,16 +97,41 @@ If you decide to opt out, Sass files will not be treated as CSS Modules by defau
 ## Keeping this fork updated
 
 We wish to keep this fork updated with the upstream repo to benefit from the ongoing open source development
-of `create-react-app`. To keep this fork up to date, please follow the steps below (based on 
-[this guide](https://robots.thoughtbot.com/keeping-a-github-fork-updated)):
+of `create-react-app`. To keep this fork up to date, please follow the steps below:
 
-```sh
-git remote add upstream git@github.com:facebookincubator/create-react-app.git
-git fetch upstream
-git rebase <commit>
-```
+1. Ensure `master` is in sync with `upstream/master`:
+
+    ```sh
+    git checkout master
+    git remote add upstream git@github.com:facebook/create-react-app.git
+    git fetch upstream
+    git pull upstream master
```

Comment:
We just want `master` to track `upstream/master` right? If so `git fetch upstream` followed by `git reset --hard upstream/master` is better than pull